app/models/dpa_exception.rb

```diff
@@ -137,7 +137,8 @@ def self.to_csv
           when 'dpa_exception_status_id'
             record.dpa_exception_status&.name
           when 'review_findings', 'review_summary', 'lsa_security_recommendation', 'lsa_security_determination', 'notes'
-            record.send(field)&.to_plain_text
+            rich_text = record.send(field)
+            rich_text.present? ? rich_text.to_plain_text : ''
           when 'tdx_tickets'
```

Comment:
I see that two weeks ago you changed the self.to_csv code in dpa_exception.rb, but did not in other models.
Why did you keep the old code in other models?
Also, the code on line 132 is not correct:
`"http://localhost:3000/dpa_exceptions/#{record.id}"`
because it uses a  localhost link. Yes, it's my three years old error but you kept it in the new code.